app/views/activities/_bounce_activity.html.erb

```diff
@@ -0,0 +1,4 @@
+<li class="activity-feed__item t-activity" id="activity-<%= bounce_activity.id %>">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+  <b>Delivery failure</b> <em><%= bounce_activity.message %></em> <em class="badge"><%= time_ago_in_words(bounce_activity.created_at) %> ago</em>
```

Comment:
That raises another point actually, there are two statuses - [`bounce` and `drop`](https://documentation.mailgun.com/api-webhooks.html#webhooks). 

Should we track both? A bounce could feasibly happen further down the line if the user's mailbox is full, whereas the failure would apply to all emails as you say.